services/local-notifications/core.go

```diff
@@ -354,3 +469,19 @@ func (s *Service) Protocols() []p2p.Protocol {
 func (s *Service) IsStarted() bool {
 	return s.started
 }
+
+func SendMessageNotifications(mnb []protocol.MessageNotificationBody) {
+	var ns []*Notification
+	for _, n := range mnb {
+		ns = append(ns, &Notification{
+			Body:     n,
+			BodyType: TypeMessage,
+			Category: "", // TODO what category do we want?
+			Deeplink: "", // TODO find what if any Deeplink should be used here
+			Image:    "", // TODO do we want to attach any image data contained on the MessageBody{}?
+		})
+	}
+
+	// sends notifications messages to the OS level application
+	pushMessages(ns)
+}
```

Comment:
@cammellos @vkjr the `Notification` struct has a couple of fields that look helpful for a building the OS notification, I've asked some questions in the code would you mind give your opinion on these?

Reply:
"Category" could be `newMessage`? and deep link I will let you know, we can sort it out at a later stage. Image similarly, for now it would be enough just to display text messages.